File: trunk/2_iteracion/Implementacion/SAGRES2.0/src/GestionPedidos/ElementoPedido.java
/*
 * To change this template, choose Tools | Templates
 * and open the template in the editor.
 */

package GestionPedidos;

import GestionCarta.Elemento;

/**
 *
 * @author Gaspar
 */
public class ElementoPedido {

    private String comentario;
    private Integer estado;
    private Integer codElementoPedido;
    private Elemento elemento;

    public ElementoPedido(String comentario){
        this.comentario = comentario;
        this.estado = 0;
        //TODO Como asignar el codigo del elementopedido
        //this.codElementoPedido = ;
    }
    public ElementoPedido(Integer codElementoPedido, Integer estado, String comentario){
        this.codElementoPedido = codElementoPedido;
        this.estado = estado;
        this.comentario = comentario;
    }
    public Integer getCodElementoPedido() {
        return codElementoPedido;
    }

    public String getComentario() {
        return comentario;
    }

    public Integer getEstado() {
        return estado;
    }
    public Elemento getElemento(){
        return elemento;
    }
    public void setComentario(String comentario) {
        this.comentario = comentario;
    }

    public void setEstado(Integer estado) {
        this.estado = estado;
    }
    
    public void asocia(Elemento ele){
        elemento = ele;
    }
}

File: 3_iteracion/Implementacion/WebSAGRES/ElementoPedido.php
<?php

/**
 * Description of ElementoPedido
 *
 * @author Adrián Víctor Pérez Lopera
 */
class elementoPedido {
    private $codElementoPedido, $comentario, $estado, $elemento;

    public function __construct($codElementoPedido, $comentario, $estado, $elemento) {
        $this->codElementoPedido = $codElementoPedido;
        $this->comentario = $comentario;
        $this->estado = $estado;
        $this->elemento = $elemento;
    }

    public function  getId() {
        return $this->codElementoPedido;
    }

    public function  getComentario() {
        return $this->comentario;
    }

    public function  getEstado() {
        return $this->estado;
    }

    public function  getElemento() {
        return $this->elemento;
    }
}

?>
